### PR TITLE
less: update to 563

### PR DESCRIPTION
**Pull request by balrog-nona**

### CODE REVIEW

File: components/text/less/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2016 <contributor>
+# Copyright 2020 <contributor>
 #
```
Comment (AndWac) [inline components/text/less/manifests/sample-manifest.p5m]: Usually we don't update the sample-manifest if only the date changes.

File: components/text/less/Makefile [hunk 2]
```diff
@@ -22,20 +22,21 @@
 #
 # Copyright (c) 2011, 2015, Oracle and/or its affiliates. All rights reserved.
 # Copyright (c) 2019, Michal Nowak
+# Copyright (c) 2020, Nona Hansel
 #
 
 PREFERRED_BITS=		64
```
Comment (AndWac) [inline components/text/less/Makefile:29]: Can you please also change to the new style:
Replace
PREFERRED_BITS=64
with
BUILD_BITS=64

File: components/text/less/Makefile [hunk 3]
```diff
@@ -22,20 +22,21 @@
...
 COMPONENT_SUMMARY=	Pager program similar to more
 COMPONENT_PROJECT_URL=	http://www.greenwoodsoftware.com/less/
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_HASH= \
-	sha256:ff165275859381a63f19135a8f1f6c5a194d53ec3187f94121ecd8ef0795fe3d
+	sha256:ce5b6d2b9fc4442d7a07c93ab128d2dff2ce09a1d4f2d055b95cf28dd0dc9a9a
 COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)$(COMPONENT_ARCHIVE)
 COMPONENT_FMRI=		text/less
 COMPONENT_CLASSIFICATION=	Development/GNU
```
Comment (AndWac) [inline components/text/less/Makefile:42]: And later replace the 
include $(WS_MAKE_RULES)/prep.mk
include $(WS_MAKE_RULES)/configure.mk
include $(WS_MAKE_RULES)/ips.mk

with
TEST_TARGET= $(NO_TESTS)
include $(WS_MAKE_RULES)/common.mk

and finally remove the old targets:
build:		$(BUILD_64)
install:	$(INSTALL_64)
test:		$(NO_TESTS)

**Review by AndWac — CHANGES_REQUESTED** [review]
My proposed changes are documented in
docs/current-tasks.md

File: components/text/less/manifests/sample-manifest.p5m (hunk 1 above)
Comment (balrog-nona) [reply components/text/less/manifests/sample-manifest.p5m]: Changed it back, thanks.

File: components/text/less/Makefile (hunk 2 above)
Comment (balrog-nona) [reply components/text/less/Makefile:29]: Changed it.

File: components/text/less/Makefile (hunk 3 above)
Comment (balrog-nona) [reply components/text/less/Makefile:42]: Replaced and removed, thank you.

**Review by AndWac — APPROVED** [review]

**Comment by AndWac** [comment]
Thank you for your contribution!

**Comment by balrog-nona** [comment]
Hopefully, I will contribute more. I'm still new with OI.

**Comment by AndWac** [comment]
Great! Feel free to ask for information or help on IRC or oi-dev mailing list.

**Pull request merged**